modules/secret/variables.tf

```diff
@@ -17,6 +17,12 @@ variable "service-account" {
   type        = string
 }
 
+variable "additional-service-account-accessors" {
+  description = "Additional service accounts that will need to access the secret."
+  type = list(string)
+  default = []
+}
+
```

Comment:
should we name this `service-accounts`, and note `service-account` is deprecated?